roles/setup_lvms/templates/lvmcluster.yml.j2

```diff
@@ -0,0 +1,18 @@
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: "{{ sl_lvm_cluster_name }}"
+  namespace: "{{ sl_lvm_namespace }}"
+spec:
+  storage:
+    deviceClasses:
+      - name: "{{ sl_dev_class_name }}"
+        default: true
+        fstype: "{{ sl_fstype }}"
+        deviceSelector:
+          paths:
+            - "{{ sl_dev_selector_path }}"
```

Comment:
I wonder if you could add in here:  `forceWipeDevicesAndDestroyAllData: {{ sl_force }}`[0] as an option to force cleanup and avoid the ssh/rsh/debug in the nodes? wdyt?

[0] https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/storage/persistent-storage-using-local-storage#explanation-of-fields-in-the-lvmcluster-cr